src/doc/tutorials/Aneurysm.rst

```diff
@@ -328,14 +329,14 @@ coordinates of this node to seed a set of streamlines.
 Plotting streamlines of velocity
 """"""""""""""""""""""""""""""""
 
-1. Go to *Add->Pseudocolor->operators->IntegralCurve->velocity*.
+1. Go to *Add->Pseudocolor->operators->IntergralCurve->velocity*.
```

Comment:
There is a typo here. "IntergralCurve" should be "IntegralCurve"/

Reply:
thanks, i fixed!